### ISSUE TITLE
Only admins can see duplicate files

### ISSUE THREAD

**Issue by mehdibo** [PaulLereverend/NextcloudDuplicateFinder#24]
When going to the Duplicate Finder tab with an admin account it works fine,
But with other accounts, it just keeps spinning, and in the console there is a 403 Request error:

```
Request URL: https://xxxxxx/apps/duplicatefinder/files
Request Method: GET
Status Code: 403 
```

**Comment by fig2k4** [comment]
Same for me..  https://github.com/PaulLereverend/NextcloudDuplicateFinder/issues/1#issuecomment-787077014

**Comment by PaulLereverend** [comment]
Fixed 42f6cf7 (v0.0.6)
